telemetry/client.go

```diff
@@ -56,6 +57,8 @@ func (c *Client) PushReceivedMessages(filter transport.Filter, sshMessage *types
 	_, err := c.httpClient.Post(url, "application/json", bytes.NewBuffer(body))
 	if err != nil {
 		c.logger.Error("Error sending message to telemetry server", zap.Error(err))
+	} else {
+		c.logger.Debug("Successfully pushed received messages to telemetry server")
 	}
```

Comment:
For the success scenarios, I'd say we need to consider whether logging here is required, due to geth.log already containing a lot of noise!.

Reply:
Removed in latest commit